src/sentry/api/endpoints/accept_organization_invite.py

```diff
@@ -25,13 +26,17 @@
 from sentry.types.region import RegionResolutionError, get_region_by_name
 from sentry.utils import auth
 
+logger = logging.getLogger(__name__)
+
 
 def get_invite_state(
     member_id: int,
     organization_slug: Optional[str],
     user_id: int,
 ) -> Optional[RpcUserInviteContext]:
     if organization_slug is None:
+        logging.info("organization.member_invite.no_slug", {"member_id": member_id})
```

Comment:
```suggestion
        logger.info("organization.member_invite.no_slug", extra={"member_id": member_id})
```

I'm not too familiar with anything special we do with our logger so I may be wrong but I think `*args` is used for formatting